ReactNativeClient/lib/string-utils.js

```diff
@@ -264,6 +264,15 @@ function substrWithEllipsis(s, start, length) {
 	return `${s.substr(start, length - 3)}...`;
 }
 
+function closestWordIndices(s, match, offset) {
+	const begin = s.slice(0, (match.index >>> 0) + 1).search(/\S+$/);
+	let end = s.slice(match.index + match[0].length + offset).search(/\s/);
+	if (end < 0) end = s.length;
+	else end += match.index + match[0].length + offset;
+
+	return [begin, end];
+}
```

Comment:
Please add a comment to describe:
- What this function does
- What are the parameters
- What is the output

Reply:
I've changed this function to nextWhitespaceIndex(s, begin) which seeks the next whitespace character from `begin` index in string `s` and return its index. It's much more simpler. Let me know if I should make any changes.